src/Libraries/DynamoUnits/DynamoUnits.Migrations.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<migrations>
+  <priorNameHint>
+    <oldName>DynamoUnits.SIUnit.Add@double</oldName>
```

Comment:
@hlp Not sure I like adding new math functions just to support migration. I understand that you can't migrate a zero touch library to an instance of a node model, but there's got to be a better way. We can't have additional redundant math operations in the library.